.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
why `-`?

Reply:
Dunno. Maybe husky has added it automatically. Should we remove this?

Reply:
From looking at the rest of the files, looks like husky uses that folder location to run the pre-commit script so lets keep.